agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/init/ConfigOverride.java

```diff
@@ -71,11 +71,17 @@ static Config getConfig(Configuration config) {
     if (!config.instrumentation.springScheduling.enabled) {
       properties.put("otel.instrumentation.spring-scheduling.enabled", "false");
     }
+    if (!config.preview.instrumentation.apacheCamel.enabled) {
+      properties.put("otel.instrumentation.apache-camel.enabled", "false");
```

Comment:
this is off by default.. should check for enabled and then set it to true?

Reply:
DisabledByDefaultInstrumentation is disabled by default. If it's true, this object doesn't seem right.

Reply:
I'm not following, can you make a specific suggestion?